_data/accessibility-tests/combo-box.yml

```diff
@@ -0,0 +1,103 @@
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 3.2.2
+# Zoom/screen magnification tests
+  - summary: Using zoom does not obstruct the combo box. 
+    summary_additional: When you zoom to 400%, you can see the combo box input and initial dropdown options. It does not overlap with other content and no scrolling is needed to see the dropdown has opened.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.10
+# Keyboard navigation tests
+  - summary: Focus indicator is clearly visible around combo box.
+    summary_additional: When you tab into a combo box, a visible outline or other clear indication of focus appears around the combo box and options within.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.7
+  - summary: Tabbing into the combo box doesn't trigger a change of context.
+    summary_additional: When using a keyboard to access the combo box, there is no change until you activate with enter key or spacebar.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.1
+  - summary: Keyboard actions can access all combo box functionality.
+    summary_additional: You can type, select, clear input and perform all other functions in a combo box element using only your keyboard.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.1.1
+# Screen reader tests
+  - summary: Screen reader announces type and number of options.
+    summary_additional: When interacting with the combo box with a screen reader, you hear the component's label, the option names, and how many options there are.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.1
+  - summary: Screen reader announces "combo box" upon initial interaction. 
+    summary_additional: When you tab onto a combo box with a screen reader, you hear "combo box" along with instructions needed to complete the combo box.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.5
+  - summary: Screen readers identify combo boxes the same way when more than one is on a page.
+    summary_additional: When you tab to a combo box on a page that has other combo boxes, you will hear "combo box" every time you tab to a new combo box.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.4
+  - summary: Combo box offers visual and auditory instructions.
+    summary_additional: When you use the combo box component, you hear details about the purpose of the input with a screen reader and see them on the page. The instructions are specific to the form's intent (e.g., "Select a state", "Select a fruit").
+    test_status: exception
```

Comment:
@amycole501 @alex-hull 

Wanting to confirm that this should be marked as "passed with exceptions". If it is, can you draft a summary of the issue we can use on the website and also open an issue in GitHub so we can link to it here?

Reply:
Technically it passes. However it's very verbose. @mejiaj mentioned some of that wordiness is coming from the combo box element (HTML 5). If that's the case I don't know if there's anything we can do to fix it. I'd be OK marking it as "pass" and drafting an issue to investigate the wordiness further. It's not a blocker per se, just a bit annoying.

Reply:
Updated status to "pass" in [4f3b5eb](https://github.com/uswds/uswds-site/pull/2811/commits/4f3b5eb761e78b74b29afae97526d41e332c05c4)